face_box[] faces
uint16 ref_img_width
uint16 ref_img_height
